### DISCUSSION TITLE
Which container should we use: nordvpn or nordlynx?

### DISCUSSION THREAD

**Discussion by imthenachoman** [bubuntux/nordvpn#369] [Q&A] [answered]
bubuntux has two contaniers. I am not sure which is the lastest one to use if we want NordVPN over WireGuard?

- https://github.com/bubuntux/nordvpn
- https://github.com/bubuntux/nordlynx

**Comment by imthenachoman** [comment]
@bubuntux Any help on this? Which is the container to use?

**Reply by bubuntux** [reply]
is up to you. 
Some ppl still use nordvpn and they are trying to keep it up to date and add features, nordlynx is lighter and simpler, i no longer use any of them and i do not maintain them, i move away to protonvpn and created a new container for my use

**Reply by imthenachoman** [reply]
I see. Is there somewhere we can see the core differences? I can't tell the difference in the two.

**Reply by bubuntux** [reply, marked as answer]
tldr; 
nordvpn uses the official nordvpn binary, it sounds great but that brings a lot of problems with iptables and maintenance so i created nordlynx.
nordlynx uses wireguard, so it simpler and lightweight
the code is open for both so feel free to take a look at them and choose your poison

**Reply by imthenachoman** [reply]
I see. I was wondering why not use the official nordvpn binary cause it seems like it would be so much easier but I guess not. This is helpful. Thank you!
